src/resource/solidDataset.ts

```diff
@@ -1156,14 +1156,43 @@ function resolveLocalIrisInThing(
 }
 
 /**
- * Fetch the contents of '.well-known/solid' for a given resource URL.
+ * @hidden
+ *
+ * Fetch a SolidDataset containing information about the capabilities of the
+ * storage server that hosts the given resource URL. For more information,
+ * please see the [ESS
+ * Documentation](https://docs.inrupt.com/ess/latest/services/discovery-endpoint/#well-known-solid).
```

Comment:
I think I would just add a note to specify that this is an ESS-specific feature, which is implied by linking to the ESS documentation but may not be explicit enough.

Reply:
It's actually not though — but this method is hidden from the docs, so these docs are really only for our benefit.